examples/pennant-rzvernal/pennant.yml

```diff
@@ -0,0 +1 @@
+pnt: 'sedovbig.pnt'
```

Comment:
change this to '/PENNANT/test/sedovbig/sedovbig.pnt'
So the user won't have to copy the file to their workdir